src/settings.rs

```diff
@@ -404,8 +404,18 @@ impl Settings {
                     settings.results = results;
                 }
 
-                settings.fuzzy =
-                    search_matches.is_present("fuzzy") || env::var("MCFLY_FUZZY").is_ok();
+                if let Ok(fuzzy) = env::var("MCFLY_FUZZY") {
+                    if let Ok(fuzzy) = i16::from_str(&fuzzy) {
+                        settings.fuzzy = fuzzy;
+                    } else {
+                        settings.fuzzy = 2;
```

Comment:
I think this needs to be something like (in pseudocode) `if fuzzy.to_lowercase() != "false" { settings.fuzzy = 2; }`

Reply:
oops, you're right -- fixed.